src/main/resources/application.yml

```diff
@@ -0,0 +1,37 @@
+logging:
```

Comment:
I believe we use 2 spaces for yml files. I would check other projects and make them consistent.